react-native-gutenberg-bridge/android/src/main/java/org/wordpress/mobile/WPAndroidGlue/WPAndroidGlueCode.java

```diff
@@ -579,33 +586,42 @@ public void toggleEditorMode(boolean htmlModeEnabled) {
     }
 
     public void appendUploadMediaFile(final int mediaId, final String mediaUri, final boolean isVideo) {
-       if (isMediaUploadCallbackRegistered() && mMediaPickedByUserOnBlock) {
-           String mediaType = getMediaType(isVideo);
-           mMediaPickedByUserOnBlock = false;
-           List<RNMedia> mediaList = new ArrayList<>();
-           mediaList.add(new Media(mediaId, mediaUri, mediaType));
-           mPendingMediaUploadCallback.onUploadMediaFileSelected(mediaList);
-       } else {
-           // we can assume we're being passed a new image from share intent as there was no selectMedia callback
-           sendOrDeferAppendMediaSignal(mediaId, mediaUri, isVideo);
-       }
+//       if (isMediaUploadCallbackRegistered() && mMediaPickedByUserOnBlock) {
+//           String mediaType = getMediaType(isVideo);
+//           mMediaPickedByUserOnBlock = false;
+//           List<RNMedia> mediaList = new ArrayList<>();
+//           mediaList.add(new Media(mediaId, mediaUri, mediaType));
+//           mPendingMediaUploadCallback.onUploadMediaFileSelected(mediaList);
+//       } else {
+//           // we can assume we're being passed a new image from share intent as there was no selectMedia callback
+//           sendOrDeferAppendMediaSignal(mediaId, mediaUri, isVideo);
+//       }
```

Comment:
Is it intentional to leave commented code here? :)

Reply:
Indeed this is no longer needed :smiley: . Since the dependent tests have been refactored, I have removed the method chain.